instrumentation/instasarama/instasarama.go

```diff
@@ -4,3 +4,55 @@
 // Package instasarama provides Instana tracing instrumentation for
 // Kafka producers and consumers build on top of github.com/Shopify/sarama.
 package instasarama
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+const KafkaHeaderEnvVarKey = "INSTANA_KAFKA_HEADER_FORMAT"
+
+type KafkaHeaderType int
```

Comment:
What do you think if instead of using int here, we will use string type? 


This will allow us to avoid adding extra type and conversion logic. All the validations and default values handling can be implemented inside "GetKafkaHeaderFormat".